src/Components/HorizontalLine.jsx

```diff
@@ -0,0 +1,12 @@
+const HorizontalLine = ({ width }) => {
+    return ( 
+        <hr style={{
+            width: width, // adjust the width as needed
+            border: "1px solid #E47347",
+            marginTop: "10px",
+            marginBottom: "10px",
```

Comment:
Alternatively you can set margin: 10px 0 which equal to 10px (top) 0 (right) 10px (bottom) 0 (left)